ZelBack/src/services/appsService.js

```diff
@@ -99,6 +99,8 @@ let masterSlaveAppsRunning = false;
 
 const hashesNumberOfSearchs = new Map();
 
+const appsThatMightBeUsingOldGatewayIpAssignment = ['HNSDoH', 'dane', 'fdm', 'Jetpack2', 'fdmdedicated', 'isokosse', 'ChainBraryDApp', 'health', 'ethercalc'];
```

Comment:
TODO recheck these apps so we dont have to hardcode it at all.

Reply:
Kamil checked and almost are using componentName for networking, but I left that there just in case...